content/css/concepts/display/display.md

```diff
@@ -46,6 +52,139 @@ The following values can be appplied to the `display` property:
 - `initial`
 - `inherit`
 
+## Differences between display values
+The display property in CSS allows you to control how elements are rendered on a webpage. Here are the commonly used display values and their descriptions:
+
+### 1. Display: block
+
+ Elements with display: block; are rendered as block-level elements. They create a line break after the element and occupy the full width of their parent container.
+```css
+.block-element {
+  display: block;
+}
+```
+
+### 2. Display: inline
+Elements with display: inline; are rendered as inline elements. They don't create line breaks and occupy only the necessary width and height based on their content.
+```css
+.inline-element {
+  display: inline;
+}
+```
+
+### 3. Display: inline-block
+
+  Elements with display: inline-block; are rendered as inline-level block elements. They behave like inline elements but allow you to set width, height, padding, and margins.
+```css
+.inline-block-element {
+  display: inline-block;
+}
+```
+### 4. Display: none
+
+ Elements with display: none; are not rendered and are completely hidden from the page. They don't take up any space in the document flow.
+```css
+.hidden-element {
+  display: none;
+}
+```
+### 5. Display: flex
+
+ Elements with display: flex; create a flex container, allowing you to build flexible and responsive layouts. Flex items inside the container can be positioned and resized using flexbox properties.
+```css
+.flex-container {
+  display: flex;
+}
+```
+### 6. Display: grid
+
+Elements with display: grid; create a grid container that enables you to define rows and columns, creating a two-dimensional grid system. Child elements inside the container can be placed in specific cells and adjusted for size and alignment.
+```css
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+```
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Overriding display values:
+
+CSS specificity allows you to override default display values of elements. Here's an example of overriding display:
+
+```css
+/* Original CSS */
+.element {
+  display: inline;
+}
+
+/* Overriding with higher specificity */
+.container .element {
+  display: block;
+}
+
+```
+
+## Responsive design considerations:
+The display property can be combined with media queries to create responsive designs. Here's an example:
+
+```css
+@media (max-width: 768px) {
+  .element {
+    display: none; /* Hide element on smaller screens */
+  }
+}
+
+@media (min-width: 768px) {
+  .element {
+    display: block; /* Display element on larger screens */
+  }
+}
+
+
+```
+
+## display: grid and display: flex comparison:
+CSS Grid and Flexbox are powerful layout tools. Here's an example comparing the two:
+
+```css
```

Comment:
Can you provide an image illustrating this?

Reply:
Yes, I want to add this image, together with a proper description and the code.

![Illustrating display flex versus display grid](https://github.com/Codecademy/docs/assets/113944962/d44dc6b4-590c-4bea-b6cc-5593b8de9809)


I checked the documentation to make sure I am using the correct format and still my image is not being displayed. The image is located in the media folder and I inserted it using 

_![Display Flex Versus Display Grid Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-display-flex-versus-grid.png)_


Still is not being displayed. Can you please help me understand what am I doing wrong? Thank you!

Reply:
@Alexandra-Haynes You've linked it in properly. It won't display right now because it's not merged. Once merged, the image link will work and it will be displayed on the page.

Reply:
That makes sense! Thank you for the fast response, I appreciate your help